go/vt/vtgate/planbuilder/testdata/vindex_func_cases.txt

```diff
@@ -329,30 +329,5 @@
   }
 }
 
-# limit and order by null clauses
```

Comment:
my understanding is that we still support `order by null`, so why has this testcase been deleted?

Reply:
It previously ignored order by null (and rand), which was kind of incorrect, especially for rand. So, I decided to instead go with the simplified rule that you shouldn't order by anything for vindex functions.